### PR TITLE
Adding onboarding track classes in Dashboard & Builder

### PR DESCRIPTION
**Pull request by xavijam**
Adding necessary changes in Dashboard and Builder for onboarding tracking in FS and GA.
Related tickets: #12912 #12906.

1. ~Sign up link~ it is not needed as we discussed yesterday.
2. Sign up button on /signup page: done in https://github.com/CartoDB/cartodb-central/pull/1883.
3. Submit button on /confirmation page: done in https://github.com/CartoDB/cartodb-central/pull/1883.
4. New map button on Dashboard: added, with class `track-onboarding--newMap` for FS, and GA event added, check code.
5. Create map button on Add datasets modal window: added, with class `track-onboarding--createMap` for FS, and GA event added, check code.
6. Edit your map button on Builder's Welcome screen: added, with class `track-onboarding--editMap` for FS, and GA event added, check code.

@noguerol @arianaescobar

### CODE REVIEW

**Comment by xavijam** [comment]
@arianaescobar, could you take a look please 💋 ?

**Comment by arianaescobar** [comment]
of course :)

File: lib/assets/core/javascripts/cartodb3/components/onboardings/builder/builder-view.js [hunk 1]
```diff
@@ -93,6 +94,15 @@ module.exports = CoreView.extend({
 
   _close: function () {
     this._checkForgetStatus();
+
+    GAPusher({
+      eventName: 'send',
+      hitType: 'event',
+      eventCategory: 'Edit Map',
```
Comment (arianaescobar) [inline lib/assets/core/javascripts/cartodb3/components/onboardings/builder/builder-view.js]: Use the exact copy please <3, "Edit your Map"

File: lib/assets/javascripts/cartodb/dashboard/views/filters.jst.ejs [hunk 2]
```diff
@@ -91,7 +91,7 @@
           <button class="
             CDB-Button CDB-Button--primary
             <%- !isMaps && !canCreateDatasets ? 'is-disabled' : ''  %>
-            <%- isMaps ? 'js-new_map' : 'js-new_dataset' %>
+            <%- isMaps ? 'js-new_map track-onboarding--newMap' : 'js-new_dataset' %>
```
Comment (arianaescobar) [inline lib/assets/javascripts/cartodb/dashboard/views/filters.jst.ejs]: This should be in the onboarding step, the welcoming screen of the dashboard when you log into Builder for the first time.

File: lib/assets/javascripts/cartodb/common/dialogs/create/create_footer.js [hunk 3]
```diff
@@ -101,6 +101,14 @@ module.exports = cdb.core.View.extend({
   },
 
   _createMap: function() {
+    GAPusher({
+      eventName: 'send',
+      hitType: 'event',
+      eventCategory: 'Create Map',
+      eventAction: 'click',
+      eventLabel: 'Dashboard'
```
Comment (arianaescobar) [inline lib/assets/javascripts/cartodb/common/dialogs/create/create_footer.js]: Let's change it to 'Add dataset modal'

File: lib/assets/core/javascripts/cartodb3/components/onboardings/builder/builder-view.js (hunk 1 above)
Comment (xavijam) [reply lib/assets/core/javascripts/cartodb3/components/onboardings/builder/builder-view.js]: :+1:

File: lib/assets/javascripts/cartodb/common/dialogs/create/create_footer.js (hunk 3 above)
Comment (xavijam) [reply lib/assets/javascripts/cartodb/common/dialogs/create/create_footer.js]: 👍

File: lib/assets/javascripts/cartodb/dashboard/views/filters.jst.ejs (hunk 2 above)
Comment (xavijam) [reply lib/assets/javascripts/cartodb/dashboard/views/filters.jst.ejs]: Chaning it right now!

**Comment by xavijam** [comment]
Feedback applied @arianaescobar 💋 .

**Comment by xavijam** [comment]
deploying it to ded11, for testing it just in case.

**Comment by xavijam** [comment]
Tested in staging, found a problem but it is now solved. Merging and deploying when all checks are passed 🚀 .

**Comment by xavijam** [comment]
Frontend tests passing, merging!

**Pull request merged**